src/components/Filters/StatefulFilters.tsx

```diff
@@ -127,8 +123,10 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
     this.updateActiveFilters(activeFilters);
   };
 
-  selectFilterType = (filterType: FilterType) => {
+  selectFilterType = value => {
```

Comment:
This is more a question than a request of a change. Why did you need to remove the type? Is there a way to easily add the type here?

Reply:
nah, we can add :string